thriftbp/client_middlewares.go

```diff
@@ -14,6 +14,21 @@ import (
 	"github.com/reddit/baseplate.go/tracing"
 )
 
+// WithDefaultRetryFilters returns a list of retrybp.Filters by appending the
+// given filters to the "default" retry filters:
+//
+// 1. ContextErrorFilter - do not retry on context cancellation/timeout.
+// 2. UnrecoverableErrorFilter - do not retry errors marked as unrecoverable with
+//    retry.Unrecoverable.
+// 3. PoolExhaustedFilter - do retry on clientpool.PoolExhausted errors.
```

Comment:
nit: Add blank lines between ordered list, or they will be rendered into the same line in godoc :(

You should be able to check locally via command:

```
go doc github.com/reddit/baseplate.go/thriftbp.WithDefaultRetryFilters
```